core/src/main/java/nl/nn/adapterframework/unmanaged/PollGuard.java

```diff
@@ -67,8 +68,14 @@ public void run() {
 				} finally {
 					log.warn("JMS poll timeout ["+pollTimeoutNr+"] handling restarting receiver ["+springJmsConnector.getListener().getReceiver().getName()+"]");
 					springJmsConnector.getListener().getReceiver().startRunning();
-					log.warn("JMS poll timeout ["+pollTimeoutNr+"] handling restarted receiver ["+springJmsConnector.getListener().getReceiver().getName()+"]");
+					if (springJmsConnector.getReceiver().isInRunState(RunState.STARTED)) {
```

Comment:
Are you sure the state must be STARTED? Please check

Reply:
I changed the check to log an error on `RunState.EXCEPTION_STARTING`.